libs/external-router/src/token/type-resolvable-routes.token.ts

```diff
@@ -20,21 +21,21 @@ export const DAFF_EXTERNAL_ROUTER_ROUTES_RESOLVABLE_BY_TYPE = new InjectionToken
 /**
  * A provider used to connect a type to a route to enable dynamic route resolution at runtime.
  */
-export function daffProvideRouteResolvableByType(
-  type: DaffExternalRouteType,
-  route: DaffRouteWithoutPath,
-): Provider {
+export function daffProvideRouteResolvableByType(typeRoutePair: DaffTypeRoutePair): Provider {
   return {
     provide: DAFF_EXTERNAL_ROUTER_ROUTES_RESOLVABLE_BY_TYPE,
     multi: true,
-    useValue: { type, route },
+    useValue: typeRoutePair,
   };
 }
 
-export function provideRoutesResolvableByType(routes: DaffTypeRoutePair[]): Provider[] {
+/**
+ * A multi-provider used to connect an array of types an respective route.
```

Comment:
I think there are some mistakes in this doc. Are you trying to say `array of types and respective routes`?

Reply:
745a55d1de1dd9dfc276766595d0b314beac6563